apps/web/src/server/views/applications/case-documentation/documentation-folder.njk

```diff
@@ -183,21 +183,33 @@
 									</td>
 								</tr>
 								{% for documentationFile in documentationFiles.items %}
-									{% set isDisabled = documentationFile.status === 'awaiting_virus_check' or documentationFile.status === 'failed_virus_check' %}
+									{% set isDisabled = documentationFile.status === 'awaiting_upload' or documentationFile.status === 'awaiting_virus_check' or documentationFile.status === 'failed_virus_check' %}
 
 									<tr class="govuk-table__row {{ "govuk-table__row--disabled" if isDisabled else "" }}">
 										<td class="govuk-table__cell">
-											{{ govukCheckboxes({
-												name: "selectedFilesIds[]",
-												items: [
-													{
-														value: documentationFile.guid,
-														text: documentationFile.documentName
-													}]
-											}) }}
+											{% if not isDisabled %}
+												{{ govukCheckboxes({
+													name: "selectedFilesIds[]",
+													items: [
+														{
+															value: documentationFile.guid,
+															text: documentationFile.documentName
+														}]
+												}) }}
+											{% else %}
+												<div class="govuk-form-group">
+													<div class="govuk-checkboxes" data-module="govuk-checkboxes">
```

Comment:
add aria-hidden="true" to the DIV so screen readers will ignore this as it's only for decoration